contrib/logging/fluentd-sidecar-es/logging-sidecar-example.yaml

```diff
@@ -14,6 +14,10 @@ spec:
       mountPath: /var/log
   - name: sidecar-log-collector
     image: gcr.io/google_containers/fluentd-sidecar-es:1.1
+    resources:
+      limits:
+        cpu: 100m
+        memory: 200Mi
```

Comment:
I find all these setting mysterious and it would be good to have them documented / justified somewhere?